test_combat.py

```diff
@@ -0,0 +1,45 @@
+from unittest.mock import Mock
```

Comment:
What is this class supposed to do?

Reply:
[I made a video on this when I was younger](https://www.youtube.com/watch?v=smPbDqGjFAI).